src/sentry/static/sentry/app/views/settings/components/forms/richListField.jsx

```diff
@@ -0,0 +1,231 @@
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import {t} from 'app/locale';
+import DropdownAutoComplete from 'app/components/dropdownAutoComplete';
+import DropdownButton from 'app/components/dropdownButton';
+import InputField from 'app/views/settings/components/forms/inputField';
+import InlineSvg from 'app/components/inlineSvg';
+import Confirm from 'app/components/confirm';
+
+const ItemList = styled('div')`
+  display: flex;
```

Comment:
I would actually recommend using display grid here, with auto flow and grid-gap, you'll reduce a lot of the spacing styling you need in `Item` and I think can also probably completely get rid of `ItemLabel`.

Reply:
Wait, `ItemLabel` does not interact with `ItemList`. I removed some redundant rules now, they were leftovers from other experiments.

FWIW: The `Item` styles, including the label, are mostly copied from our `Button` implementation, which I could massage into rendering like this. I'd rather keep it close to how Button is structured.